README.md

```diff
@@ -617,6 +617,36 @@ isRoot | boolean | no | This option defines where the certificate should be inst
 
 The content of the generated .mobileconfig file as base64-encoded string. This config might be useful for debugging purposes. If the certificate has been successfully set via CLI then nothing is returned.
 
+### mobile: listCertificates
+
+Lists installed certificates for real devices only if [py-ios-device](https://github.com/YueChen-C/py-ios-device) tool is available on the server machine since driver version 4.10.0.
+
+#### Returned Result
+
+Returns map of certificates installed on the real device. The response looks like:
+
+```
```

Comment:
please add `json` after backticks , so github could highlight it properly

Reply:
Add `json` after backticks